controllers/controller.js

```diff
@@ -69,3 +70,14 @@ exports.postCommentsByArticleId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+  removeCommentById(comment_id)
+    .then(() => {
+      res.status(204).send();
```

Comment:
Good job not sending anything back here - a 204 doesn't allow us to do so anyways, so nicely done :-)